src/app/slider/slider-sample-1/slider-sample-1.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+    <label igxLabel class="titleLabel">Price Range Filter</label>
+</div>
+<div class="container">
+    <igx-slider id="slider" [type]="sliderType.RANGE" [minValue]="0" [maxValue]="1000" [lowerBound]="100" [upperBound]="900" [(ngModel)]="priceRange"></igx-slider>
+    <div class="lowerInput">
+      <label igxLabel for="lowerInput">Lower price:</label>
+      <input igxInput id="lowerInput" type="text" [ngModel]="priceRange.lower | currency" (blur)="updatePriceRange($event.target)" />
+    </div>
```

Comment:
We have a wrapper class for the input controls. We could use this class to have an input that feels "material". Something like that:
```html
<div class="igx-form-group lowerInput">
        <input igxInput name="lowerInput" type="text" [ngModel]="priceRange.lower | currency" (blur)="updatePriceRange($event.target)">
        <label igxLabel for="lowerInput">Lower price:</label>
 </div>
```